### PR TITLE
first pr for task 10

### PR DESCRIPTION
**Pull request by Jay-R-Mac**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -309,3 +310,30 @@ describe("DELETE /api/comments/:comment_id", () => {
...
+      .then((data) => {
+        expect(Array.isArray(data.body)).toBe(true);
+      });
+  });
+  it("responds with an array of users which contains a username, name and avatar URL", () => {
+    return request(app)
+      .get("/api/users")
+      .then(({ body }) => {
+        expect(body.length).toBe(4);
+        body.forEach((user) => {
```
Comment (B7Bprog) [inline __tests__/app.test.js]: Users should arrive on a key of 'users' in an object, not directly on the 'body'. Please refactor this.

File: controller/controller.js [hunk 2]
```diff
@@ -88,6 +89,12 @@ const selectComment = function (req, res, next) {
       next(err);
     });
 };
+
+const sendUsers = function (req, res, next) {
+  getUsers().then((users) => {
+    res.status(200).send(users);
```
Comment (B7Bprog) [inline controller/controller.js]: Please respond with an object where the users array is on a key of 'users'.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hi Jordan, there's one thing to refactor (see comments), then you can merge it.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -322,12 +322,12 @@ describe("GET /api/users", () => {
         expect(Array.isArray(data.body)).toBe(true);
       });
   });
-  it("responds with an array of users which contains a username, name and avatar URL", () => {
+  it.only("responds with an array of users which contains a username, name and avatar URL", () => {
```
Comment (NRMSMITH) [inline __tests__/app.test.js]: make sure to not push up .onlys

**Review by NRMSMITH — APPROVED** [review]
All good - just make sure not to push up .onlys :)

File: __tests__/app.test.js (hunk 3 above)
Comment (Jay-R-Mac) [reply __tests__/app.test.js]: I changed this a minute later you were just too fast for me!

**Pull request merged**